src/buildApplication/processVercelOutput.ts

```diff
@@ -22,8 +23,11 @@ export async function getVercelStaticAssets(): Promise<string[]> {
 		return [];
 	}
 
-	return (await readPathsRecursively(dir)).map(file =>
-		addLeadingSlash(normalizePath(relative(dir, file)))
+	return (
+		(await readPathsRecursively(dir))
+			.map(file => addLeadingSlash(normalizePath(relative(dir, file))))
+			// Filter out the worker script output directory contents as those are not valid static assets.
+			.filter(path => !/^\/_worker\.js\//.test(path))
```

Comment:
I don't think that files inside the `_worker.js` directory are uploaded as assets, so this is probably non-necessary? 🤔 (not that it hurts anything either 🤔)

Reply:
They are collected as assets and injected into the worker file in the list of different assets/functions available. Filtering this list is just trimming down a little bit of the unnecessary fat in the worker index.js file.

Reply:
yeah sorry, wrangler's totally unrelated here! my bad 🙇